packages/application-shell/src/components/navbar/main-navbar.styles.ts

```diff
@@ -0,0 +1,514 @@
+  left: ${NAVBAR.sublistIndentationWhenCollapsed};
+  z-index: -1;
+  list-style: none;
+  position: fixed;
+  display: none;
+`;
+
+const ScrollableMenu = styled.div`
+  flex: 1 1 0;
+  overflow-x: hidden;
+  overflow-y: hidden;
+  padding: var(--spacing-30) var(--spacing-30) ${NAVBAR.itemSize};
+  width: ${NAVBAR.widthLeftNavigation};
+  box-sizing: border-box;
+  /* For Firefox */
+  scrollbar-width: thin;
+  scrollbar-color: var(--color-primary-40) transparent;
+
+  :hover {
+    overflow-y: scroll;
+    padding-right: 8px;
+  }
+
+  ::-webkit-scrollbar {
+    width: 8px;
+  }
+
+  ::-webkit-scrollbar-track {
+    background: transparent;
+  }
+
+  ::-webkit-scrollbar-thumb {
+    background-color: var(--color-primary-40);
+    border-radius: var(--border-radius-8);
+  }
+`;
+
+const FixedMenu = styled.div`
+  position: relative;
+  width: ${NAVBAR.widthLeftNavigation};
+`;
+
+const TextLink = styled.a`
+  color: var(--color-for-navbar-link);
+  text-decoration: none;
+  display: flex;
+  padding: var(--spacing-25);
+  align-items: center;
+  justify-content: center;
+`;
+
+const SupportMenu = styled.div`
+  padding: var(--spacing-10) var(--spacing-30) var(--spacing-20)
+    var(--spacing-30);
+  height: calc(var(--item-size) + var(--spacing-20));
+`;
+
+const Title = styled.div`
+  flex: 1;
+  font-weight: 600;
+  font-size: var(--font-size-for-navbar-link);
+  line-height: var(--line-height-for-navbar-link);
+  color: var(--color-for-navbar-link);
+  opacity: 0;
+  transition: ${NAVBAR.leftNavigationTransition};
+  text-align: left;
+  text-decoration: none;
+  z-index: 1;
+`;
+
+const Text = styled.div`
+  font-weight: var(--font-weight-for-navbar-link);
+  font-size: var(--font-size-for-navbar-link);
+  line-height: var(--line-height-for-navbar-link);
+  width: 100%;
+  height: 100%;
+`;
+
+const getMenuItemLinkStyles = (isSubmenuLink: boolean) => [
+  isSubmenuLink &&
+    css`
+      display: -webkit-box;
+      -webkit-line-clamp: 2;
+      -webkit-box-orient: vertical;
+      color: var(--color-solid);
+      font-weight: var(--font-weight-for-navbar-link-when-hovered);
+      text-decoration: none;
+      flex: 1;
+      padding: var(--spacing-25) var(--spacing-25) var(--spacing-25)
+        var(--spacing-30);
+      transition: padding 150ms ease-out;
+    `,
+  !isSubmenuLink &&
+    css`
+      color: var(--color-for-navbar-link);
+      text-decoration: none;
+      display: flex;
+      padding: var(--spacing-25);
+      align-items: center;
+      justify-content: center;
+    `,
+];
+
+export {
+  getMenuItemLinkStyles,
+  leftNavigationOpenStyles,
+  listStyles,
+  sublistStyles,
+  // styled components
+  Expander,
+  ExpanderIcon,
+  FixedMenu,
+  HeaderTitle,
+  Icon,
+  IconContainer,
+  ItemIconText,
+  LeftNavigation,
+  NavigationHeader,
+  NavlinkClickableContent,
+  ScrollableMenu,
+  SublistItem,
+  SupportMenu,
+  Text,
+  TextLink,
+  TextLinkSublistWrapper,
+  Title,
+  Tooltip,
+  TooltipContainer,
+};
```

Comment:
SInce this is a huge file and it seems it has components/helpers for both the `navbar` and the `menu-items` files, I was wondering if we could also split this file into two so the code would be more manageable. 🤔

Reply:
Thank you, split in 2c075e7d3a035a0be54744383abbde18cd0adf93